packages/flame/lib/src/game/camera/camera_component.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:ui';
+import '../../../components.dart';
+import 'camera.dart';
+
+/// Do not use.
+///
+/// This class encapsulates BaseGame's rendering functionality. It will be
+/// converted into a proper Component in a future release, but until then
+/// using it in any code other than from the BaseGame class is unsafe and
+/// therefore not recommended.
+///
+/// "Co" means unfinished Component.
+class CameraCo {
```

Comment:
I think I would prefer to have this as `CameraController` and remove the comments about not using this outside base game. In the past implementation, camera was possible to be used on the `Game` class, so this is basically the same.

Also I would name this file `camera_controller` instead of component.

Reply:
Unfortunately, `CameraController` would not be a proper name for this class, since it doesn't actually control the camera. Currently, the logic for controlling the camera lives inside the `Camera` class itself.

The purpose of this class is that it's a Camera component (i.e. it enables the use of Camera class within the Flame Component System), except that we downgraded it from an actual Component into a not-quite-component for now, because it is not yet ready to be used just about anywhere a Component might be used.

Reply:
We could name it only camera then, like it was in the past. I understand that this will became a component later, but probably for the next release, it won't. So it would better have a proper name and file name and then the comment above wouldn't be needed anymore.

Then on a future change, it could easily be promoted again to a component

Reply:
But we already have a class called `Camera`, so we can't call this one `Camera` too. I guess we could call it `Camera2`, or `CameraWrapper`, or?

Reply:
For sure not `Camera2`.. Getting android flashbacks here

Reply:
CameraWrapper sounds good to me!

Reply:
ok, renamed once again